packages/clay-charts/src/Geomap.js

```diff
@@ -263,20 +259,9 @@ Geomap.STATE = {
 		selected: '#4b9bff',
 		value: 'pop_est',
 	}),
-
-	/**
-	 * Geo-json data
-	 * @instance
-	 * @memberof Geomap
-	 * @type {?Function|?Object|?String}
-	 * @default undefined
-	 */
-	data: Config.oneOfType([
-		Config.func(),
-		Config.object(),
-		Config.string(),
-	]).required(),
 };
 
+Object.assign(Geomap.STATE, DataComponent.STATE);
```

Comment:
Since we already extend `DataComponent`, is this really necessary?

Reply:
No, you're right.